cmake/32-bit-toolchain.cmake

```diff
@@ -33,6 +33,8 @@ endif()
 set (GCC_PREFIX i686-w64-mingw32)
 set (CMAKE_C_COMPILER ${GCC_PREFIX}-gcc)
 set (CMAKE_CXX_COMPILER ${GCC_PREFIX}-g++)
+set (CMAKE_AS_COMPILER as)
```

Comment:
Is the AS naming intended here (see below) ?

Reply:
After some thought, I will drop both of these changes (and below). The Windows builds don't use the assembly, and the supercop check should still work with Windows.

Reply:
Removed here and below.